tuna.go

```diff
@@ -745,6 +750,21 @@ func (c *Common) CreateServerConn(force bool) error {
 				return err
 			}
 
+			entryToExitMaxPrice, exitToEntryMaxPrice, err := ParsePrice(c.ServiceInfo.MaxPrice)
+			if err != nil {
+				return err
+			}
+			if entryToExitMaxPrice > 0 || exitToEntryMaxPrice > 0 {
+				balance, err := c.Wallet.Balance()
+				if err != nil {
+					return err
```

Comment:
I think we shouldn't return when failed to get balance so tuna won't be affected by on-chain incidents.

Reply:
Yes, modified.